templates/extra/Dockerfile.TS.sern

```diff
@@ -8,6 +8,6 @@ RUN npm install
 
 COPY . .
 
-RUN tsc --build
+RUN tsc --build || npx tsc --build
```

Comment:
add -p flag maybe to let tsc use local tsconfig?

```tsc -p "./tsconfig.json" --build || npx tsc -p "./tsconfig.json" --build

Reply:
doesnt it choose automatically?

Reply:
I'm doing this because most of the times `tsc` isn't installed automatically in the dockerfile.

Reply:
from my experience not adding -p flag messes up build